RxCocoa/iOS/Proxies/RxPickerViewDataSourceProxy.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  RxPickerViewDataSourceProxy.swift
+//  RxCocoa
+//
+//  Created by Sergey Shulga on 05/07/2017.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+#if os(iOS) || os(tvOS)
+
+    import UIKit
+#if !RX_NO_MODULE
+    import RxSwift
+#endif
+
+let pickerViewDataSourceNotSet = PickerViewDataSourceNotSet()
+
+final class PickerViewDataSourceNotSet: NSObject, UIPickerViewDataSource {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        rxAbstractMethod()
```

Comment:
I believe we should present nothing when data source isn't set. It seems to me that asserting would crash the app in case somebody first observed some appropriate method and then set data source.